tracks/bash/exercises/hamming/mentoring.md

```diff
@@ -0,0 +1,147 @@
+### Reasonable Solutions
```

Comment:
Feel free to grab more from https://github.com/IsaacG/exercism/blob/master/mentoring/bash ;)

Reply:
Thanks, happy to make other changes just trying to get the docs created.